kedro-datasets/tests/ibis/test_table_dataset.py

```diff
@@ -86,11 +74,6 @@ def test_exists(self, table_dataset, dummy_table):
         table_dataset.save(dummy_table)
         assert table_dataset.exists()
 
-    @pytest.mark.parametrize("load_args", [{"filename": True}], indirect=True)
```

Comment:
I think this test should be kept, but the `load_args` passed can be something different, like `{"database": "test"}` (after #909).

Reply:
Hi @deepyaman , I tried adding below test, but it keeps failing saying `Catalog "random" does not exist!`. I am not sure what I am missing. Could you please help me in resolving the issue and suggest what else I can assert for the `load_args` test. Thank you

```
@pytest.mark.parametrize("load_args", [{"database": "random.db"}], indirect=True)
    def test_load_extra_params(
        self, dummy_table, tmp_path, load_args
    ):
        """Test overriding the load arguments."""
        db_path = tmp_path / load_args["database"]
        table_dataset = TableDataset(table_name="test", load_args=load_args)
        con = duckdb.connect(str(db_path))
        table_dataset.save(dummy_table)
        table_dataset.load()
     
        assert (
            "test" in con.sql("SELECT * FROM duckdb_views").fetchnumpy()["schema_name"]
        )
```